hydra/compose.py

```diff
@@ -45,13 +46,12 @@ def compose(
 
     if strict is not None:
         # DEPRECATED: remove in 1.2
-        warnings.warn(
+        deprecation_warning(
             dedent(
-                """\
-
-        The strict flag in the compose API is deprecated and will be removed in the next version of Hydra.
-        See https://hydra.cc/docs/upgrades/0.11_to_1.0/strict_mode_flag_deprecated for more info.
-        """
+                """
+                The strict flag in the compose API is deprecated and will be removed in the next version of Hydra.
+                See https://hydra.cc/docs/upgrades/0.11_to_1.0/strict_mode_flag_deprecated for more info.
+                """
```

Comment:
This is changing the printed string.
Can you trigger the deprecation and see what it actually looks like before and after?

Reply:
I believe you are mistaken, the string is not changed :)
```python
>>> msg1 = dedent(
...                  """\
...
...         The strict flag in the compose API is deprecated and will be removed in the next version of Hydra.
...         See https://hydra.cc/docs/upgrades/0.11_to_1.0/strict_mode_flag_deprecated for more info.
...         """
... )
>>> msg1
'\nThe strict flag in the compose API is deprecated and will be removed in the next version of Hydra.\nSee https://hydra.cc/docs/upgrades/0.11_to_1.0/strict_mode_flag_deprecated for more info.\n'
>>> msg2 = dedent(
...                 """
...                 The strict flag in the compose API is deprecated and will be removed in the next version of Hydra.
...                 See https://hydra.cc/docs/upgrades/0.11_to_1.0/strict_mode_flag_deprecated for more info.
...                 """
... )
>>> msg2
'\nThe strict flag in the compose API is deprecated and will be removed in the next version of Hydra.\nSee https://hydra.cc/docs/upgrades/0.11_to_1.0/strict_mode_flag_deprecated for more info.\n'
>>> assert msg1 == msg2
```
I took away the backslash at the beginning of the string, so the extra newline is no longer necessary.

Reply:
I triggered the warning (using the `compose` `strict` flag) just to be sure.

Reply:
Interesting, I tested myself and got a different string, but maybe me copying the input caused the string to change.